### PR TITLE
Feat: send new user

### PR DESCRIPTION
**Pull request by Draeken**

### CODE REVIEW

File: src/utils.ts [hunk 1]
```diff
@@ -19,9 +19,13 @@ export function checkFileChanges(context: ExtensionContext): void {
 }
 
 export function createUserId(context: ExtensionContext) {
-  if (!context.workspaceState.get('userId')) {
-    context.workspaceState.update('userId', uniqid());
+  let userId = context.workspaceState.get('userId') as string | undefined;
+  if (!userId) {
+    console.log('new user!');
```
Comment (guillaumebarranco) [inline src/utils.ts]: remove console log please

File: src/utils.ts [hunk 2]
```diff
@@ -19,9 +19,13 @@ export function checkFileChanges(context: ExtensionContext): void {
 }
 
 export function createUserId(context: ExtensionContext) {
-  if (!context.workspaceState.get('userId')) {
-    context.workspaceState.update('userId', uniqid());
+  let userId = context.workspaceState.get('userId') as string | undefined;
```
Comment (guillaumebarranco) [inline src/utils.ts:22]: why the casting ?

File: src/config.ts [hunk 3]
```diff
@@ -0,0 +1 @@
+export const config = { apiUrl: 'http://b61690c1.ngrok.io' };
```
Comment (guillaumebarranco) [inline src/config.ts]: should not be commited !!!

File: src/utils.ts (hunk 2 above)
Comment (Draeken) [reply src/utils.ts:22]: default type for context.workspaceState.get('userId')  is {} | undefined, but it's a string, not an object 😮

File: src/utils.ts (hunk 1 above)
Comment (Draeken) [reply src/utils.ts]: it's done 😮

File: src/config.ts (hunk 3 above)
Comment (Draeken) [reply src/config.ts]: :rage4:

**Review by guillaumebarranco — APPROVED** [review]

**Pull request merged**